### PR TITLE
Refactoring the Greetings screen

### PR DESCRIPTION
**Pull request by ezisazis**
There was an unused menu resource which got deleted with every related files.

+1 removed some unused resources
+1 removed a single comment

### CODE REVIEW

**Comment by sinisterstuf** [comment]
## Danger has errored
[!] Invalid `Dangerfile` file: No such file or directory @ rb_sysopen - app/build/reports/lint/lint-result.xml
```
 #  from Dangerfile:7
 #  -------------------------------------------
 #  # Linting
 >  android_lint.lint
 #  commit_lint.check
 #  -------------------------------------------
```

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="http://danger.systems/">Danger</a>
</p>

**Comment by ezisazis** [comment]
screen shot 2017-08-29 at 0 16 13

Danger should not say all green, cuz it's not :D

**Comment by ezisazis** [comment]
@sinisterstuf don't merge it yet, I will need to remove `action_help` resource from the strings.xml too!

**Comment by ezisazis** [comment]
@sinisterstuf done, ready to merge

File: app/src/main/java/org/sinisterstuf/guesstheanimal/Greeting.java [hunk 1]
```diff
@@ -1,34 +1,24 @@
 package org.sinisterstuf.guesstheanimal;
 
-import android.os.Bundle;
 import android.app.Activity;
 import android.content.Intent;
-import android.view.Menu;
+import android.os.Bundle;
```
Comment (sionleroux) [inline app/src/main/java/org/sinisterstuf/guesstheanimal/Greeting.java:5]: I was confused by this addition at first and now I see you've just re-ordered the imports.

**Review by sionleroux — APPROVED** [review]
This menu item (which I even translated to 3 other languages) was meant to have instructions on the game, explaining how it works, but I never got round to making it. Anyway on modern Android there is no menu key, so if we do end up doing this we should probably add it to a concealed menu accessible from the bar at the top.

Thanks for cleaning up!

**Comment by sionleroux** [comment]
I won't wait for coveralls, it's not going to finish because it can't start because there are no tests.

**Comment by ezisazis** [comment]
We should create a `Main menu` for the app and inside of it the usual things like `Play game` / `How to play`  etc...

**Comment by sionleroux** [comment]
IMO it's not bad to throw the user straight into the game when you start, for this app, because the first question kind of guides you through it (hence I never wrote a help page), but it would still be good to have a menu with Options, About & Help, or similar. We could make a new ticket for that.

**Pull request merged**